chirp/chirp_common.py

```diff
@@ -929,7 +929,6 @@ def __init__(self):
         self.init("has_variable_power", False,
                   "Indicates the radio supports any power level between the "
                   "min and max in valid_power_levels")
-
```

Comment:
Please don't make unrelated changes, especially just whitespace damage. I'm sure this was accidental, but it needs to be unchanged.

Reply:
pep8 burped on this during my local tox testing, which is probably why I made changes.